pymc_marketing/mmm/plot.py

```diff
@@ -262,6 +262,13 @@ def plot_curve(
         Figure and the axes
 
     """
+    scalar_coords_to_drop = []
+    for coord, values in curve.coords.items():
+        if values.size == 1 and coord not in curve.indexes:
+            scalar_coords_to_drop.append(coord)
+
+    for coord in scalar_coords_to_drop:
+        curve = curve.reset_coords(coord, drop=True)
```

Comment:
Can this be done in a single line? I believe the reset_coords takes a list

Reply:
Hello, 
Yes indeed it seems that the reset_coords can receive a list as well, pushed the change...